### PR TITLE
fix: align mobile navigation to header

### PR DESCRIPTION
**Pull request by hwkr**
Google Chrome-2023-12-13-22-40-22@2x
Google Chrome-2023-12-13-22-40-17@2x

### CODE REVIEW

**Comment by amritk** [comment]
Wow sick will check it tomorrow

File: packages/api-reference/src/components/ApiReferenceLayout.vue [hunk 1]
```diff
@@ -363,7 +368,7 @@ const { state } = useApiClientStore()
     position: absolute;
 
     /* Offset by 1px to avoid gap */
-    top: calc(var(--refs-header-height) - 1px);
+    top: -1px;
```
Comment (amritk) [inline packages/api-reference/src/components/ApiReferenceLayout.vue:371]: why do we need this? like why is there a gap?

**Review by amritk — APPROVED** [review]
Looks good! Works great

**Review by marclave — APPROVED** [review]
awesome work @hwkr

**Comment by hanspagel** [comment]
Let’s get this in! :)

**Pull request merged**